TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>Also, in a phase-folded plot, we would expect to see just one "dip" per planet. Seeing multiple "dip"s here is an indication of either multiple planets, or (as we know) an error in the data processing </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>good to know, changed this example anyway. </p>